web/gallery-pearl.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
```

Comment:
What about changing the javascript to load the JSON file obtained by replacing `.html` with `.json`.  Then we could symlink this to the base gallery, and avoid code duplication.

Your choice, though.